taccsite_cms/static/site_cms/css/src/_imports/objects/o-offset-content.css

```diff
@@ -27,8 +26,8 @@ Styleguide Objects.OffsetContent
 	.o-offset-content--left + .o-offset-content--left { clear: left; }
 }
 @media only screen and (--medium-and-above) and (--max-wide-and-below) {
-  [class*="c-offset-content--"] {
-    --max-width: 50%;
+  [class*="o-offset-content--"] {
+    max-width: 50%;
   }
```

Comment:
**Explanation.** Make style effectual (change unused variable to actual style, fix classname typo).